python/ray/serve/_private/controller.py

```diff
@@ -260,7 +260,7 @@ def record_autoscaling_metrics(self, data: Dict[str, float], send_timestamp: flo
         )
         self.deployment_state_manager.record_autoscaling_metrics(data, send_timestamp)
 
-    def record_handle_metrics(self, data: Dict[str, float], send_timestamp: float):
+    def record_handle_metrics(self, data, send_timestamp: float):
```

Comment:
please replace the type hint

Reply:
I'm passing a big mix of ids and request metrics here, the type hints will be very messy. I will add a follow-up PR that refactors the metrics pusher to use kwargs instead of args for the callback function, so we can avoid shoving everything into this `data` parameter.